docs/pages/machine-id/reference/gitlab.mdx

```diff
@@ -79,12 +83,46 @@ spec:
         # ref restricts joins to jobs that were triggered by a specific git
         # reference. Combine this with `ref_type` to create allow rules that
         # can only be triggered by a specific branch or tag.
+        #
+        # This field supports simple "glob-style" matching:
+        # - Use '*' to match zero or more characters.
+        # - Use '?' to match any single character.
         ref: main
         # sub is a single string that concatenates the project_path, ref_type
         # and ref. This can be used to restrict joins using a single string,
         # whilst also describing a specific project and git ref.
         #
         # It is better to use the individual fields, as it is easy to mis-format
         # the sub string.
+        #
+        # This field supports simple "glob-style" matching:
+        # - Use '*' to match zero or more characters.
+        # - Use '?' to match any single character.
         sub: project_path:my-user/my-project:ref_type:branch:ref:main
+        # user_login restricts joins to jobs that were triggered by a specific
+        # username.
+        user_login: octocat
+        # user_email restricts joins to jobs that were triggered by a specific
+        # user with the given email
+        user_email: octo.cat@example.com
+        # ref_protected if set to true restricts joins to jobs running against a
+        # protected ref.
+        # If omitted, the protection status of the ref is not checked.
+        ref_protected: true
+        # environment_protected if set to true restricts joins to jobs running
+        # against a protected ref.
+        # If omitted, the protection status of the ref is not checked.
+        environment_protected: true
+        # ci_config_sha restricts joins to jobs that are using a specific
+        # commit of CI configuration.
+        ci_config_sha: ff4713bc047d87bf7eac9674765ae793478c50ff
```

Comment:
Should we use a more obviously fake value here?

Reply:
Sure - I can make this all `f` and that might be clearer ?

Reply:
613a1c2c7b